miner/payload_building_test.go

```diff
@@ -208,26 +216,43 @@ func holoceneConfig() *params.ChainConfig {
 	return &config
 }
 
-// newPayloadArgs returns a BuildPaylooadArgs with the given parentHash and eip-1559 params,
-// testTimestamp for Timestamp, and testRecipient for recipient. NoTxPool is set to true.
-func newPayloadArgs(parentHash common.Hash, params1559 []byte) *BuildPayloadArgs {
+func jovianConfig() *params.ChainConfig {
+	config := *params.TestChainConfig
+	config.LondonBlock = big.NewInt(0)
+	t := uint64(0)
+	config.CanyonTime = &t
+	config.HoloceneTime = &t
+	config.JovianTime = &t
+	canyonDenom := uint64(250)
+	config.Optimism = &params.OptimismConfig{
+		EIP1559Elasticity:        6,
+		EIP1559Denominator:       50,
+		EIP1559DenominatorCanyon: &canyonDenom,
+	}
+	return &config
```

Comment:
could we just do
```suggestion
	config := holoceneConfig()
	config.JovianTime = ptr(uint64(0)) // or whatever uint64 ptr function this test package offers...
```
?

Reply:
Certainly [11c2240](https://github.com/ethereum-optimism/op-geth/pull/666/commits/11c2240e19fa3b6685779ad308e7deaea1e60a06)